_studio/mfx_lib/cmrt_cross_platform/include/cmrt_cross_platform.h

```diff
@@ -70,6 +73,16 @@ class SurfaceIndex
 private:
     unsigned int index;
 #ifdef CM_LINUX
+    /*
+     * Do not delete this line:
+     * SurfaceIndex is commonly used as CM kernel function's parameter.
+     * It has virutal table and has copy constructor, so GNU calling convetion will pass the object's pointer to kernel function.
+     * This is different with Windows VC++, which always copies the entire object transferred on the callee's stack.
+     *
+     * Depending on the special object size after adding below "extra_byte",
+     * SetKernelArg and SetThreadArg can recognize this object and follow GNU's convention to construct kernel function's stack.
+     * Detail can be found on "http://wiki.ith.intel.com/display/VPGCompiler/CM+Linux"
```

Comment:
Please, remove the link from the comment.

Reply:
@dvrogozh removed